README.md

```diff
@@ -25,6 +25,11 @@ This repository holds the kcp tests that tests against the publicly available  i
   ```console
   $ kubectl oidc-login get-token --oidc-issuer-url=<oidc issuer url> --oidc-client-id=<oidc client ID> --oidc-redirect-url-hostname=127.0.0.1
   ```
+  - For kcp services deploy with redhat sso( **Only supported kcp services deploy with redhat sso** ), there's another login method by use `offline token`, you could get an offline token ( [How to generate an offline token](https://access.redhat.com/articles/3626371#bgenerating-a-new-offline-tokenb-3) ) and log in to the kcp service by execute the follow commands.
```

Comment:
s/deploy/deployed
Do not think we need that text in Bold Letters
s/there's another login method by use/there's another login method called `offline token` that could be used.
s/execute the follow commands/executing the following commands with out actually needing to input credentials into the browser.

Reply:
Hmm, bold Letters may be not necessary  and update the descriptions, addressed.